redaxo/src/core/lib/setup/import.php

```diff
@@ -237,7 +237,7 @@ private static function reinstallPackages()
         if ($error) {
             $error = '<ul class="rex-ul1">
             <li>
-            <h3 class="rex-hl3">' . rex_i18n::msg('setup_513', '<span class="rex-error">', '</span>') . '</h3>
```

Comment:
ändert sich hier nun die visuelle darstellung?

Reply:
Die Params werden in der Message gar nicht mehr verwendet, daher nein.